src/maintenance/repositories/official/index.md

```diff
@@ -0,0 +1,13 @@
+# Official Repositories
+
+Void provides other official repositories, which are maintained by the Void
+project, but not installed in the default configuration:
+
+- [debug](./debug.md): contains debugging symbols for packages
+- [multilib](./multilib.md): contains 32-bit libraries for 64-bit systems (glibc
+   only)
+- [multilib](./multilib.md)/nonfree: contains non-free multilib packages
```

Comment:
Intentional that this link looks like this?

Reply:
Yes, but I will rework how the multilib/nonfree repository is represented.